xbmc/addons/GUIDialogAddonInfo.cpp

```diff
@@ -86,6 +87,15 @@ bool CGUIDialogAddonInfo::OnMessage(CGUIMessage& message)
       }
       if (iControl == CONTROL_BTN_INSTALL)
       {
+        if (m_localAddon)
+        {
+          if (m_localAddon->Type() == ADDON_ADSPDLL && ActiveAE::CActiveAEDSP::Get().IsProcessing())
+          {
+            CGUIDialogOK::ShowAndGetInput(24077, 0, 24085, 0);
+            return true;
+          }
+        }
+
```

Comment:
@tamland 
thanks for pointing out this issue. I found the missing labels in an older version.